salt/swap/init.sls

```diff
@@ -5,6 +5,7 @@ coreutils:
 unmount-swaps:
   cmd.run:
     - name: /sbin/swapoff -a
+    - onlyif: test `tail -n +2 /proc/swaps | wc -l` != 0
```

Comment:
Small nitpick this should goes as other PR

Reply:
This is for making `unmount-swaps` idempotent: it will not try to `swapoff` if there are no swap partitions.

Reply:
Ok, I've got your point